js/src/components/main-tab-nav/main-tab-nav.js

```diff
@@ -8,12 +8,13 @@ import { getNewPath, getPath } from '@woocommerce/navigation';
  * Internal dependencies
  */
 import { glaData } from '~/constants';
+import useGoogleMCAccount from '~/hooks/useGoogleMCAccount';
 import AppTabNav from '~/components/app-tab-nav';
 import useMenuEffect from '~/hooks/useMenuEffect';
 import GtinMigrationBanner from '~/components/gtin-migration-banner';
 import { getShippingUrl } from '~/utils/urls';
 
-let tabs = [
+const ALL_TABS = [
```

Comment:
@ankitrox I would leave the `tabs` variable as is (not a constant) since you are not using `ALL_TABS` again. You are reassigning the `ALL_TABS` constant to the `tabs` variable.

Reply:
Restored the `tabs` variable.